keycloak-event-emitter/src/main/java/io/cloudtrust/keycloak/kafkaeventemitter/KafkaEventEmitterState.java

```diff
@@ -0,0 +1,37 @@
+package io.cloudtrust.keycloak.kafkaeventemitter;
+
+public class KafkaEventEmitterState {
+    private String state;
+
+    public void initialized(){
+        state = "initialized";
+    }
+
+    public  void  starting(){
+        state = "starting";
+    }
+
+    public void pending(){
+        state = "pending";
+    }
+
+    public void working(){
+        state = "working";
+    }
+
+    public boolean isInitialized(){
+        return state == "initialized";
```

Comment:
Forbidden! `"initialized".equals(this.state)` but would be better to implement this another way